src/components/vest/VestDealTokens.tsx

```diff
@@ -284,89 +296,85 @@ export const VestDealTokens: React.FC = ({ ...restProps }) => {
             </SortableTH>
           ))}
         </TableHead>
-        {!data.length ? (
```

Comment:
Maybe it will be better to move this part into separate function? To avoid code duplicating

Reply:
Good catch!